### ISSUE TITLE
xcrun error

### ISSUE THREAD

**Issue by aytanozu** [distriqt/AIR-ImageScripts#1]
I create assets for the application but I get the following error:
xcrun: error: unable to find utility "actool", not a developer tool or in PATH

**Comment by marchbold** [comment]
Have you installed Xcode and the command line tools?

**Comment by zizusoft** [comment]
Hi - I get exactly the same error.

I have both XCode and the command line tools installed

**Comment by zizusoft** [comment]
Going to Xcode > Preferences > Locations and selecting a default Command Line Tool location fixed this issue.

**Comment by marchbold** [comment]
Yeah you need to ensure the command line tools are installed and that you have pointed them to the correct version. You can do this through Xcode or using `xcode-select` on the command line.
